docs/_templates/layout.html

```diff
@@ -122,6 +122,11 @@
     {%- endfor %}
   </div>
   <div id="sidebar">
+    {% if pagename == 'api' or '/api' in pagename %}
```

Comment:
```suggestion
    {% if pagename.split("/")[-1] == "api" %}
```

Reply:
How this currently is makes it future proofed against #722.

Reply:
optimised in https://github.com/DisnakeDev/disnake/pull/963/commits/9dcffe34b474a846ceb0bd6fb218ca8c42b780b2 and kept it future proofed